doc/BrevUtsending_Aktivitetskrav_Innhenting_README.md

```diff
@@ -0,0 +1,35 @@
+# Rutine for automatisk utsendelse av brev for innhenting av aktivitetsplikt
+
+## Bakgrunn
+
+Saksbehandler må en gang i året sende ut brev til brukere som har skolegang som aktivitet 
+for å vurdere aktivitetsplikten etter endt semester.
+[Favro kort.](https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=NAV-8258)
+
+## Hvordan er automatiseringen løst?
+
+For personer som er i utdanning lages det ved vedtak om overgangsstønad en fremleggsoppgave for innhenting av
+aktivitetsplikt. I 2023 tilsvarte dette ca ~1600 oppgaver. Disse oppgavene ble flyttet av saksbehandlere til mappe 
+*64 utdanning*, med frist 17. mai. Brevmaler [todo] ligger i Sanity.
+
+### Aktivering av utsendelsflyten
+For å aktivere den automatiske brevutsendingen må man sende en `POST-request` via [swagger](https://familie-ef-sak.intern.nav.no/swagger-ui/index.html#/) mot endepunktet
+`https://familie-ef-sak.intern.nav.no/api/automatisk-brev-innhenting-aktivitetsplikt/opprett-tasks`. Man må sende
+med følgende verdier i request-bodyen:
+- **liveRun** `[true, false]`: Bruk `false` om du kun vil verifisere at koden finner riktig antall oppgaver som ligger i mappen.
+Velg `true` om du ønsker å lagre ned tasks for generering og utsending av brev basert på oppgavesøket.
+- **taskLimit** `[Int]`: Et øvre tak for hvor mange oppgaver som hentes ut.
+
+I tillegg er man nødt til å [gå inn i unleash og aktivere feature toggle for automatisk brevinnhenting av aktivitetsplikt for prod miljøet.](https://teamfamilie-unleash-web.nav.cloud.nais.io/projects/default/features/familie.ef.sak.automatiske-brev-innhenting-aktivitetsplikt)
```

Comment:
Broken link. Ser ut til at det er byttet url til gui til https://teamfamilie-unleash-web.iap.nav.cloud.nais.io/

Ref: https://nav-it.slack.com/archives/C9BPTSULS/p1716989706166429